pkg/kubelet/apis/cri/runtime/v1alpha2/api.proto

```diff
@@ -1105,8 +1105,8 @@ message UInt64Value {
 
 // StorageIdentifier uniquely identify the storage..
 message StorageIdentifier{
-    // UUID of the device.
-    string uuid = 1;
+    // Mountpoint of the image filesystem.
```

Comment:
The `StorageIdentifier` is a more generic message type. The comment should not include "image filesystem".

On the other hand, is `StorageIdentifier` still an accurate name, or should we rename it to, say, `FilesystemIdentifier`?

Reply:
Will do then.

Reply:
Done. Changed to `FilesystemIdentifier`.